src/Html.php

```diff
@@ -3851,7 +3851,7 @@ public static function initEditorSystem($id, $rand = '', $display = true, $reado
         $cache_suffix = '?v=' . FrontEnd::getVersionCacheKey(GLPI_VERSION);
         $readonlyjs   = $readonly ? 'true' : 'false';
 
-        $invalid_elements = 'applet,canvas,embed,form,object';
```

Comment:
Why remove `embed` and `object`? That honestly seems tags that should not be used.

Reply:
With `convert_unsafe_embeds=true`, they will be converted to `audio`, `img` or `video` tags. The purpose is to correctly handle pasting a content from an HTML page that uses `object` or `embed` for a media content.

Anyway, this is not mandatory, and cannot be done before https://github.com/tinymce/tinymce/issues/9516 is fixed on TinyMCE side.